com.bitwig.BitwigStudio.yaml

```diff
@@ -0,0 +1,77 @@
+---
+app-id: com.bitwig.BitwigStudio
+command: bitwig-studio
+runtime-version: "20.08"
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+rename-desktop-file: bitwig-studio.desktop
+rename-icon: bitwig-studio
+finish-args:
+  # TODO: we need MIDI, Audio, DRI, and realtime performances
+  - --device=all
+  - --nodevice=kvm
+
+  # pipewire
+  - --filesystem=xdg-run/pipewire-0
+
+  # Needed to talk with pulseaudio or pipewire
```

Comment:
technically this is also needed for x11, for the sake of the being accurate

Reply:
So what should I add?

Reply:
just update the comment.
`# Needed to talk with x11, pulseaudio or pipewire`

TBF I'm not even sure it's needed for pipewire or pulsaudio. Definitely for x11 in addition to the socket.

Reply:
Done.
Thank you for the suggestion :)